cypress/integration/TC04_programs.spec.js

```diff
@@ -0,0 +1,12 @@
+let LOCAL_URL = 'http://localhost:3000/programs';
```

Comment:
⛏️ Not sure you need this? I thought you could do `/programs`

Reply:
nah just tried, it assumes /programs is a file. I think the http:// and the full url is required. If not, the tests still pass with let LOCAL_URL = 'http://localhost:3000/programs';